FEWD/Week03/lesson05/homework/week3-relaxr-part2/css/style.css

```diff
@@ -0,0 +1,211 @@
+/*
+Project Name: Blank Template
+Client: Your Client
+Author: Your Name
+Developer @GA in NYC
+*/
+
+
+/******************************************
+/* SETUP
+/*******************************************/
+
+/* Box Model Hack */
+* {
+ box-sizing: border-box; /* IE */
+}
+
+/* Clear fix hack */
+.clearfix:before, .clearfix:after {
+  content: "";
+  display: table;
+}
+
+.clearfix:after{
+  clear: both;
+}
+
+.alignright {
+  float: right;
+  padding: 0 0 10px 10px; /* note the padding around a right floated image */
+}
+
+.alignleft {
+  float: left;
+  padding: 0 10px 10px 0; /* note the padding around a left floated image */
+}
+
+/******************************************
+/* BASE STYLES
+/*******************************************/
+
+body {
+  background-color: #f0efef;
+  color: #000;
+  font-size: 12px;
+  line-height: 1.4;
+  font-family: 'Open Sans', Helvetica, Arial, sans-serif;
+  width: 100%;
+}
+
+
+/******************************************
+/* LAYOUT
+/*******************************************/
+
+header {
+  background-color: #033048;
+  height: 80px;
+}
+
+h1 {
+  font-size: 40px;
+  color: white;
+  font-weight: 700;
+  padding: 10px 0 0 70px;
+  float: left;
+}
+
+nav {
+  float: right;
+  padding-right: 75px;
+}
+
+nav li {
+  font-size: 18px;
+  display: inline-block;
+  padding-right: 30px;
+  padding-top: 28px;
+}
+
+nav li a {
+  text-decoration: none;
+  color: #f9e42e;
+}
+
+main {
+  background-color: #f0efef;
+  width: 80%;
+  margin: 0 auto;
+}
+
+main section {
+  float: left;
+  width: 700px;
+}
+
+.article-heading {
+  font-size: 40px;
+  color: #033048;
+  font-weight: 600;
+  padding-top: 50px;
+  padding-bottom: 20px;
+}
+
+.article-snippet {
+  padding: 20px 0;
+  font-size: 20px;
+  color: #606161;
+}
+
+article p::first-letter {
```

Comment:
Ooooooh fancy!!! Using `:first-letter` instead of a separate `<span>` in the HTML well done!! That said the double `::` is the old way of writing this, you only need a single `:`

Also, since you've given those paragraphs where you want this "large first letter"-effect a class of `article-snippet`, instead of complicating your CSS with references to two html tags, like you do above, you could just reference the class on the `<p>` instead:

`.article-snippet:first-letter { ... }`